include/exadg/fluid_structure_interaction/precice/driver.h

```diff
@@ -83,6 +83,26 @@ class Driver
   std::shared_ptr<ExaDG::preCICE::Adapter<dim, dim, VectorType>> precice;
   ExaDG::preCICE::ConfigurationParameters                        precice_parameters;
 
+  // Nomenclature
+  // time window: denotes time interval after which FSi coupling takes place
+  // subcycling: a single field solver performs several time steps until the next FSI coupling time
+  // window is reached
+  // time step size: refers to single field solvers. In case of subcycling, one
+  // time window consists of several time step sizes.
+
+  // The time-window size is determined through the preCICE configuration by the user:
+  // The <coupling-scheme.. tag has options for
+  // <time-window-size method="fixed" ... /> which refers to a constant time window size
+  // <time-window-size method="first-participant" ... /> which refers to an adaptive time window
+  // size prescribed by the first (which is usually the Fluid) participant. The latter is only
+  // applicable in serial coupling schemes.
```

Comment:
I think ExaDG developers/users need an explanation of serial/parallel coupling, because one otherwise thinks of one vs. multiple MPI ranks.

serial coupling = the single-field solvers are called sequentially/consecutively with data transfer from one field to another after in single-field solver.
parallel coupling = the single-field solvers are called simultaneously with data transfer in both directions afterwards.

I would also like taking about "sequential" and "simultaneous" coupling schemes (avoiding the serial/parallel notation).

Reply:
I understand your motivation for changing the terminology here. However, for someone dealing with preCICE (or configuring preCICE) the terminology serial/parallel coupling scheme will be inevitably. I copied over your explanation comments.